crowdin.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Your Crowdin credentials
+#
+"project_id_env" : "CROWDIN_PROJECT_ID"
+"api_token_env" : "CROWDIN_PERSONAL_TOKEN"
+"base_path" : "."
+"base_url" : "https://api.crowdin.com"
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+#
+# Files configuration
+#
+files: [
+  {
+    "source" : "/webapp/src/main/resources/locale/notification/MatrixChat_en.properties",
+
+    "translation" : "%original_path%/%file_name%!_%locale_with_underscore%.%file_extension%",
+    "translation_replace" : {YML_CROWDIN_LANGUAGES_ARG},
+    "dest" : "addons/matrix/MatrixChat.properties",
```

Comment:
I think addons is not correct, to check with @plamarque

Reply:
Yes, right :+1: 

```suggestion
    "dest" : "hub/matrix/MatrixChat.properties",
```